### ISSUE TITLE
possible conflict with auto pair

### ISSUE THREAD

**Issue by jerrywang1981** [luochen1990/rainbow#123]
### **Describe the bug**

in auto pair, or other configuration, if I write
test(){|}. -- | is the cursor, if I press enter, it will be indented.

test() {
    |
}

however, if I install our plugin, it is 
test() {
|
}

### **To Reproduce**
Steps to reproduce the behavior, **A piece of code to demo the issue is appreciated**.

### **Additional context**
**Are you using some third-party syntax plugins?** Add any other context about the problem here.

**Comment by luochen1990** [comment]
what is the file type (`:echo &ft`)?
what is your indent method?
If your indent method depends on the syntax object's name, then maybe you should disable the specific parentheses for this specific filetype.

**Comment by jerrywang1981** [comment]
you are right, I just found it worked in .vim file, but not in my html / typescript files, I installed other plugins like htm5,coc.nvim,etc. not sure which one blocked 'rainbow' from working or what else I need to configure.

**Comment by zetashift** [comment]
I have the same issue in markdown codeblocks for example
```scala
def foo() = {
|
}
```
I'm not sure if it's autopair or because it's markdown file?
